.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
-        bwc_version : [ "2.0.1", "2.1.0", "2.2.1", "2.3.0", "2.4.1", "2.5.0-SNAPSHOT" ]
+        bwc_version : [ "2.0.1", "2.1.0", "2.2.1", "2.3.0", "2.4.1", "2.5.0-SNAPSHOT", "2.6.0-SNAPSHOT" ]
```

Comment:
should we put the version with a suffix : -SNAPSHOT? Is there a better way to keep this bwc_version list clean?

Reply:
Seems like 2.5.0 is not released as of now. Let's make sure that once it is released we are removing the -SNAPSHOT suffix.

Reply:
Sounds good